content/docs/guides/reflex.md

```diff
@@ -205,10 +205,12 @@ This command applies the migration to the database, updating the schema to match
 
 ## Create a Customer Data App in Reflex with Neon
 
-Learn how to use Reflex with Neon Postgres in the following project to create an interactive Customer Data App.
+Learn how to use Reflex with Neon Postgres to create an interactive Customer Data App. The app demonstrates how to edit tabular data from a live application connected to a Postgres database. You can find a live version of the application [here](https://customer-data-app.reflex.run/).
 
 <DetailIconCards>
 
-<a href="https://github.com/reflex-dev/templates/tree/main/customer_data_app" description="Reflex with Neon Postgres" icon="github">Customer Data App</a>
+<a href="https://github.com/reflex-dev/templates/tree/main/customer_data_app" description="GitHub repository for the Reflex Customer Data App built with Neon Postgres" icon="github">Customer Data App</a>
 
 </DetailIconCards>
+
+![Reflex Customer Data App](/docs/guides/reflex_customer_data_app.png)
```

Comment:
![Screenshot 2024-07-01 at 4 18 52 PM](https://github.com/neondatabase/website/assets/46300090/d401dec2-cc44-4964-a4e3-1f3407b515e5)

How about we move the image to be just below the heading: "Create a Customer Data App in Reflex with Neon"?

Reply:
Done!